omod/src/main/webapp/resources/htmlforms/122a-HIVCare_ARTCard-SummaryPage.xml

```diff
@@ -0,0 +1,1732 @@
+                            </tbody>
+                        </table>
+                    </section>
+                    <section>
+                        <h5 style="background-color: #377F9C;color: white;">Switch to 2nd
+                            Line or Substitution within 2nd Line
+                        </h5>
+                        <table class="multi">
+                            <tbody>
+                            <repeat>
+                                <template>
+                                    <obsgroup groupingConceptId="99063">
+                                        <tr>
+                                            <td class="ARTStartEnableDisable7">
+                                                <label>Date</label>
+                                                <obs id="99164_1" conceptId="99164"/>
+                                            </td>
+                                            <td class="ARTStartEnableDisableChild7">
+                                                <label>New Regimen</label>
+                                                <span class="ArtCareEnableDisable7">
+                                    <small>
+                                        <obs conceptId="90315"
+                                             answerConceptIds="99007,99008,99044,99043,99282,99283,99046,99017,99018,99019,99045,99284,99285,99286,99046,99046,99887,99888,99144"
+                                             answerLabels="ABC-ddI(250)-LPV/r,ABC-ddI(400)-LPV/r,TDF-3TC-LPV/r,TDF-FTC-LPV/r,ZDV-ddI(250)-LPV/r,ZDV-ddI(400)-LPV/r,ZDV-3TC-LPV/r,ABC-ddI-LPV/r,ABC-ddI-NFV,ABC-ddI-SQV/r,ZDV-ddI-LPV/r,ZDV-ABC-LPV/r,ABC-ddI-ATV/r,AZT-3TC-ATV/r,AZT-3TC-LPV/r,TDF-3TC-ATV/r,ABC-3TC-ATV/r,Other"/>
+                                    </small>
+                                    </span>
+                                                <span class="ArtCareEnableDisableChild7">
+                                        <obs size="20" conceptId="99147" labelText="Other"/>
+                                    </span>
+                                            </td>
+                                            <td colspan="2" class="ArtCareEnableDisableChild7">
+                                                <label>Why</label>
+                                                <obs conceptId="90247" answerConceptIds="90046,90047,99889"
+                                                     answerLabels="6.Clinical, 7.Immuno, 8. Virological"/>
+                                            </td>
+                                            <td align="center">
+                                                <a href="#" class="add">Add</a> | <a href="#" class="remove">Remove</a>
+                                            </td>
+                                        </tr>
+                                    </obsgroup>
+                                </template>
+                                <render/>
```

Comment:
@dbaluku There is a looping mechanism for rendering the template multiple times see TB card

Reply:
Its the one I used